js/modules/k6/experimental/tracing/trace_id.go

```diff
@@ -18,8 +18,8 @@ const (
 	// To not ingest and process the related spans, b/c they are part of a non-cloud run.
 	k6LocalCode = 33
 
-	// metadataTraceIDKeyName is the key name of the traceID in the output metadata.
-	metadataTraceIDKeyName = "trace_id"
+	// MetadataTraceIDKeyName is the key name of the traceID in the output metadata.
+	MetadataTraceIDKeyName = "trace_id"
```

Comment:
nit: I am kind of against this being public and you using it.

Again this package is 99% likely to never get out of experimental and users will be able to add `trace_id` even without it.

It seems a lot more reasonable for the insights output to just document that certain metrics with the `trace_id` metadata will have their samples processed. 

Having this exported to get this string seems completely unnecessary dependancy. 

see https://go-proverbs.github.io/

Reply:
Fixed in [8860d2f](https://github.com/grafana/k6/pull/3100/commits/8860d2f4ca90adba808d7fdafeedf4165adea123).